entity.go

```diff
@@ -55,6 +53,15 @@ func (ck ClusteringKey) String() string {
 	return ck.Name + " ASC"
 }
 
+// Format a key's ClusteringKeys.
+func formatClusteringKeys(keys []*ClusteringKey) string {
+	pieces := make([]string, len(keys))
+	for index, ck := range keys {
```

Comment:
Re-phrasing my last comment: do we assume these are always sorted?

Reply:
> I'm a bit lost on how this affects dosa's operations, can you provide context on the existing behavior and the change's impact?

Absolutely no changes that any application will see. The only changes are in the details of the errors -- which have been carefully chosen so that they still use the same words that the tests check for, but for a human reading them there will be more detail about exactly what the mismatch is.

Order matters for clustering keys, but I'm not sure how that connects to this code so I wonder if I understand your question.

Reply:
> Order matters for clustering keys, but I'm not sure how that connects to this code so I wonder if I understand your question.

Got it, I was missing the context around the order here.